Map/src/main/java/dk/gruppesex/bodtrd/map/MapGenerator.java

```diff
@@ -20,54 +22,55 @@
 public class MapGenerator
 {
     private static Random _rand = new Random(); // TODO needs to be able to pick seed
-    private static int _wallAmount = 50;
-    private static int _wallSize = 50;
+    private final static int WALL_AMOUNT = 50;
+    private final static int WALL_SIZE = 50;
+    private final static File WALL_IMAGE_FILE = new File("../../../Map/src/main/java/dk/gruppesex/bodtrd/map/assets/wall_box.png");
```

Comment:
pretty sure its a really really bad idea passing around references to File handles. they typically have to be closed to release resources, otherwise the file is locked up in the filesystem and we can have possible memory leaks.